packages/react/src/ReactHooks.js

```diff
@@ -110,3 +110,8 @@ export function useImperativeMethods<T>(
   const dispatcher = resolveDispatcher();
   return dispatcher.useImperativeMethods(ref, create, inputs);
 }
+
+export function useDebugValueLabel(valueLabel: string) {
+  const dispatcher = resolveDispatcher();
+  return dispatcher.useDebugValueLabel(valueLabel);
```

Comment:
Can we wrap this in `if (__DEV__) {`? That would make this function a noop which can be trivially eliminated by a bundler. At least with ES modules. So that would let us remove the callsite itself in prod bundles.